packages/psss/src/components/DateToolbar/DateToolbar.js

```diff
@@ -121,7 +121,7 @@ export const DateToolbarComponent = ({ period, setPeriod }) => {
   const start = getFormattedStartByPeriod(period, 'MMM d');
   const end = getFormattedEndByPeriod(period, 'MMM d , yyyy');
 
-  const isNextDisabled = isFuturePeriod(addWeeksToPeriod(period, 1));
```

Comment:
IsFuture period only works if you pass in a period value which has a DAY period and here it is a WEEK period